RNAseq/Pipeline_GL-DPPD-7101_Versions/GL-DPPD-7101-F.md

```diff
@@ -2456,7 +2419,7 @@ pd.set_option('display.max_columns', None)
 print(combined)
 
 # Create metadata table containing samples and their respective ERCC spike-in Mix number
-# Sometimes Number in [Spike-in Mix Number] is spelled 'number' and this could cause error in mismatch search 
```

Comment:
Notebook did not get this comment update.
Should revise comment in notebook/DPPD for next release